src/pages/Home/index.js

```diff
@@ -0,0 +1,30 @@
+import  React from 'react'
+
+import './styles.scss'
```

Comment:
relative import should be under absolute import 
https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m